iothub_client/src/iothubtransport_amqp_cbs_auth.c

```diff
@@ -29,7 +29,7 @@ typedef struct AUTHENTICATION_INSTANCE_TAG
     ON_AUTHENTICATION_ERROR_CALLBACK on_error_callback;
     void* on_error_callback_context;
 
-    size_t cbs_request_timeout_secs;
+    time_t cbs_request_timeout_secs;
```

Comment:
It would be better not to make this change.
First because the result of difftime is not of time_t type. Second because time_t is not an arithmetical type.
And also there is no guarantee that time_t will be a time in seconds, like the name of the variable suggests.

Reply:
Changed to `uint64_t`